src/axom/sina/docs/sphinx/relationships.rst

```diff
@@ -0,0 +1,63 @@
+.. ## Copyright (c) 2017-2024, Lawrence Livermore National Security, LLC and
+.. ## other Axom Project Developers. See the top-level LICENSE file for details.
+.. ##
+.. ## SPDX-License-Identifier: (BSD-3-Clause)
+
+.. _relationships-label:
+
+=============
+Relationships
+=============
+
+Sina ``Relationship`` objects are used as a way to describe the correlation between
+two ``Record`` instances (and/or ``Record`` inheritors, e.g. ``Run``). A ``Relationship``
+consists of three parts: a subject, an object, and a predicate. The subject and object
+must be IDs referring to valid instances of ``Records``, while the predicate may be
+any string.
+
+In describing the connection between objects, a ``Relationship`` is read as "<subject>
+<predicate> <object>". For example, in the relationship "Alice knows Bob", "Alice" is
+the subject, "knows" is the predicate, and "Bob" is the object. Below are some additional
+examples:
+
+    - Task_22 contains Run_1024
+    - msub_1_1 describes out_j_1_1
+    - Carlos sends an email to Dani
+    - local_task_12 runs before local_run_14
+
+Note that a ``Relationship`` is described in the active voice. **Avoiding the passive
+voice in predicates is recommended**, as this keeps the "direction" of the relationship
+constant. An example of a passively-voiced ``Relationship`` is "Dani is emailed by Carlos".
+Instead, this should be phrased as "Carlos emails Dani".
```

Comment:
(unless I'm missing something) the following somehow feels like a passive way to state this.
> **Avoiding the passive voice in predicates is recommended**

Please clarify/tighten this paragraph, if possible.